scripts/ccpp_prebuild.py

```diff
@@ -70,7 +70,7 @@ def import_config(configfile, builddir):
     if not os.path.isfile(configfile):
         logging.error("Configuration file {0} not found".format(configfile))
         success = False
-        return
+        return(success, config)
```

Comment:
Why the tuple? Doesn't Python do that anyway? Is that a style thing and if so, do we want to be consistent?

Reply:
@gold2718 Sorry, I'm not sure what you mean. This function as called from main() expects two return values, and so currently when this block of code is executed (when there is no configuration file) it fails with a TypeError because it is not returning the expected values.

```
> ../scripts/ccpp_track_variables.py -s test_track_variables/suite_small_suite.xml -c test_track_variables/ccpp_prebuild_config -m test_track_variables/ -v air_pressure
ERROR:root:Configuration file test_track_variables/ccpp_prebuild_config not found
Traceback (most recent call last):
  File "/glade/derecho/scratch/kavulich/CCPP/track_variables_unit_test/ccpp-framework/test_prebuild/../scripts/ccpp_track_variables.py", line 217, in <module>
    track_variables(args.sdf,args.metadata_path,args.config,args.variable,args.debug)
  File "/glade/derecho/scratch/kavulich/CCPP/track_variables_unit_test/ccpp-framework/test_prebuild/../scripts/ccpp_track_variables.py", line 191, in track_variables
    (success, config) = import_config(config, None)
TypeError: cannot unpack non-iterable NoneType object
```

This change fixes the error.

Reply:
Thanks for the clarification. I did not realize this until you brought this up, but [parentheses are not strictly necessary for tuples](https://docs.python.org/3/library/stdtypes.html#tuple), so this would be purely stylistic. Since the existing return tuples use parens I'll stick with that for now, especially since PEP8 doesn't seem to specify a preference from what I can tell.